Description: do not use crossterm windows feature
 crossterm in debian does not enable the windows feature.
Author: Ananthu C V <weepingclown@debian.org>
Forwarded: not-needed
Last-Update: 2025-08-17
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -75,3 +75,2 @@
     "events",
-    "windows",
 ]
